src/AppBundle/Entity/Client.php

```diff
@@ -288,13 +303,13 @@ public function getFullname()
 
     public function setCourtDateWithoutTime($courtDate = null)
     {
-        $this->courtDate = ($courtDate instanceof \DateTime) ?
-                new \DateTime($courtDate->format('Y-m-d')) : null;
+        $this->courtDate = ($courtDate instanceof DateTime) ?
+                new DateTime($courtDate->format('Y-m-d')) : null;
```

Comment:
@slizzio I didn't initially write this, but I think it's correct. This method wants to assign a date without the time starting from the given date. An alternative could be cloning + setTime(0,0,0) http://php.net/manual/en/datetime.settime.php